xdsl/transforms/experimental/ConvertStencilToLLMLIR.py

```diff
@@ -393,18 +398,20 @@ def map_returns(op: Operation) -> None:
         apply = op.parent_op()
         assert isinstance(apply, ApplyOp)
 
-        res = list(apply.res)[0]
-
-        if (len(res.uses) > 1) or (not isinstance(
-            (store := list(res.uses)[0].operation), StoreOp)):
-            warn("Only single store result atm")
-            return
+        for res in list(apply.res):
+            if (len(res.uses) > 1) or (not isinstance(
+                (store := list(res.uses)[0].operation), StoreOp)):
+                warn("Only single store for a single return op result atm")
+                return
```

Comment:
I think we want to get rid of this warning if we support multiple returns. We should just check that they all map to `StoreOp`s still, before we have inlining / automatic buffering passes.

Reply:
For the case of inlining, it is not necessary that all results of `stencil.apply` will map to `StoreOp`. 
For ex. 
```
...

%1, %2 = stencil.apply(...) {...}

%3, %4 = stencil.apply(%1, ...) {...}

stencil.store(%1, %0) {...}

...
```

Reply:
Right, so we could get rid of this whole check and warning?